### PR TITLE
🚀 Added 3D Swiper Animation with Beautiful Colors for Testimonial Section

### LINKED ISSUE #839: [style]: Adding Relevant Content and Enhance Testimonial Section with 3D and Colorful Animations
### What's the style idea?

**Issue Description:**
We are looking to revamp our testimonial section on the website to create a more engaging and visually appealing experience for our users. The current testimonial cards lack the desired vibrancy and excitement that we want to convey about our product. To achieve this, we plan to introduce 3D animations and incorporate colorful visuals.

**Objective:**
The goal of this enhancement is to create a dynamic and attention-grabbing testimonial section that captures the essence of the positive feedback from our users.

**Proposed Changes:**
- Implement 3D animations for testimonial cards to provide an interactive experience when users hover over them.
- Incorporate vibrant and appealing color schemes that align with our branding, making the testimonials visually captivating.
- Replace the existing testimonials with more relevant and impactful ones that showcase the range of benefits our product offers.
- Utilize high-quality images associated with each testimonial to enhance the visual appeal.
- Integrate Swiper.js library for creating a slider that showcases multiple testimonials at a time, providing a seamless user experience.

**Expected Outcome:**
By implementing these changes, we aim to create an eye-catching and engaging testimonial section that highlights the positive impact of our product on our users' lives. This enhancement will not only attract attention but also contribute to building trust and credibility among potential users.

### Add screenshots

https://github.com/SurajPratap10/Imagine_AI/assets/85815172/251b8b56-d5a3-4368-b1a6-9ca45d95cdc7

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Nishitbaria**
# Description

This pull request introduces an exciting enhancement to the testimonial section of our project by integrating a stunning 3D swiper animation with vibrant colors. The testimonials are now displayed in an engaging and eye-catching manner that adds a touch of dynamism to our website.

## Fixes #839 

**Changes Made:**

- Integrated the Swiper library for creating a smooth and interactive carousel.
- Implemented a 3D animation effect that adds depth and perspective to the testimonials.
- Carefully selected a palette of beautiful colors to create a visually appealing design.
- Ensured responsive design, making the animation and colors consistent across different devices.
- Enhanced user experience by enabling seamless navigation between testimonials using intuitive swiper controls.

**Benefits:**

- **Enhanced Visual Appeal:** The addition of 3D animation and beautiful colors elevates the overall look and feel of the testimonial section, making it more visually captivating.

- **Engaging User Experience:** The swiper animation engages users with its dynamic transitions and smooth navigation, encouraging them to explore the testimonials.

- **Modern Design:** The use of modern animation techniques and color schemes aligns with current design trends, enhancing our website's modern aesthetics.

- **Improved Accessibility:** The implementation maintains accessibility standards, ensuring that all users can enjoy the enhanced testimonial section.

## Screenshots

[screen-capture.webm](https://github.com/SurajPratap10/Imagine_AI/assets/85815172/fd7db069-a3b2-4a22-895f-92a3cadc5f29)

|        BEFORE        |        AFTER         |
| :------------------: | :------------------: |
| Image | Image |

## Checklist

- [x] Tests have been added or updated to cover the changes
- [x] Documentation has been updated to reflect the changes
- [x] Code follows the established coding style guidelines
- [x] All tests are passing

### CODE REVIEW

File: views/slider.ejs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    </div>
+</section>
+
+
+
+
+
+
+<script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.1.6/js/swiper.min.js"></script>
```
[the remark this answers is not in the document]
Comment (Nishitbaria) [reply views/slider.ejs:115]: solved

File: views/slider.ejs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        </div>
+    </div>
+</section>
+
+
+
+
+
+
+<script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>
```
[the remark this answers is not in the document]
Comment (Nishitbaria) [reply views/slider.ejs:114]: its a trusted source

**Review by SurajPratap10 — APPROVED** [review]

**Comment by SurajPratap10** [comment]
Functionality is not working

**Comment by Nishitbaria** [comment]
@SurajPratap10 ,now you can review and merge my pull request

**Pull request merged**